website/content/docs/v0.14/Guides/logging.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Logging
+---
+
+## Viewing logs
+
+Kernel messages can be retrieved with `talosctl dmesg` command:
```

Comment:
this might be expanded, but I don't want to push on this PR:

* `talosctl logs` can also retrieve container logs
* we now have a way to send kernel logs to json-lines destination